app/src/routes/login/login.vue

```diff
@@ -55,9 +55,9 @@ export default defineComponent({
 
 		const appStore = useAppStore();
 
-		const providers = ref([]);
+		const providers = ref<Array<{ driver: string; name: string }>>([]);
```

Comment:
Lets use `{ driver: string; name: string }[]` here to be consistent with the rest of the codebase